handlers.go

```diff
@@ -5316,5 +5318,5 @@ func (s *Server) getShuttleConfig(hostname string, authToken string) (interface{
 }
 
 func (s *Server) isContentAddingDisabled(u *User) bool {
-	return s.CM.contentAddingDisabled || u.StorageDisabled
+	return s.CM.globalContentAddingDisabled || u.StorageDisabled
```

Comment:
This is not correct. for this case, we are checking if content can be added to this box - so `s.CM.contentAddingDisabled || u.StorageDisabled` is the correct way

Reply:
Same as above